test/redux_spec.js

```diff
@@ -139,6 +151,163 @@ describe("redux", ()=> {
     }));
   });
 
+  it("check double call second returns first", (done)=> {
+    let count = 0;
+    const rest = reduxApi({
+      test: "/test"
+    }).use("fetch", url=> {
+      const newUrl = `${url}-${count}`;
+      const timeout = count ? 100 : 200;
```

Comment:
It's not correct test. Response time for concurrent requests isn't determine. 
```
fetch({ url: "A" }).then(()=> console.log("A"));
fetch({ url: "B" }).then(()=> console.log("B"));
// Output might be "A", "B" or "B", "A"
```
Right code should be like 
```js
const timeout = Math.random() * 100; 
```
And this means, that test will be 50% failed